### ISSUE TITLE
TypeError: android.support.v7.widget.CardView is not a constructor

### ISSUE THREAD

**Issue by ghost** [bradmartin/nativescript-cardview#84]
I have tried the steps in the post mentioned in README.md, but i still get this error:
```
An uncaught Exception occurred on "main" thread.
com.tns.NativeScriptException: 
Calling js method onCreateView failed 

TypeError: android.support.v7.widget.CardView is not a constructor
```

**Comment by bradmartin** [comment]
Error indicates the support lib with the CardView class is not present in your application executable, so a build issue is present. If you're on NS 4x, it's possible this plugin needs a minor change. Haven't looked into this personally, will try this next week.

**Comment by bradmartin** [comment]
just published 3.0.0 to npm which should fix this.
